Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3024,7 +3042,7 @@ def gen2021HiMix(fragment,howMuch):
                   '--datatier':'ALCARECO',
                   '--eventcontent':'ALCARECO',
                   '--triggerResultsProcess': 'RECO'}
-steps['ALCAEXPCOSMICSRUN3']={'-s':'ALCAOUTPUT:@allForExpressCosmics,ALCA:PromptCalibProdSiStrip',
+steps['ALCAEXPCOSMICSRUN3']={'-s':'ALCAOUTPUT:@allForExpressCosmics,ALCA:PromptCalibProdSiStrip+PromptCalibProdSiPixelLAMCS',
```

Comment:
I don't know if it's worth updating it in this PR, but since you added the `ALCAPROMPT` producer in this step, you might want to add to 1005.0 also the corresponding harvester at:

https://github.com/cms-sw/cmssw/blob/f03d0562168ead49f929fca6a7451cc7e8abd595/Configuration/PyReleaseValidation/python/relval_production.py#L23